src/main/java/Duke.java

```diff
@@ -1,10 +1,81 @@
+import java.util.Arrays;
+import java.util.Scanner;
+
 public class Duke {
+    static Task[] tasks = new Task[100];
+    static final String TASK_ADDED_PREFIX = "Got it. I've added this task:\n\t";
     public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println("____________________________________________________________\n"
+                            + logo
+                            + "Hello! I'm Duke!\n"
+                            + "What I can do for you?\n"
+                            + "____________________________________________________________\n"
+                            );
+        String inputText;
+        String outputMessage;
+        while (true) {
```

Comment:
Should this method be divided? For example, mark method, unmark method can be seperated from this method.